### DISCUSSION TITLE
Window title update based on active tab

### DISCUSSION THREAD

**Discussion by VelliusG** [BornToBeRoot/NETworkManager#1436] [Ideas]
Have the application update the window title to reflect which tabs are selected.

Ex:
NETworkManager - Remote Control - PROFILENAME
NETworkManager - Web Console - PROFILENAME
NETworkManager - Web Console - PROFILENAME - URL

This would allow someone to use Auto-Type features from KeePass to input credentials based on current tabs selected in the application.

**Comment by BornToBeRoot** [comment]
Lets continue here #1437
